iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergOutputCommitter.java

```diff
@@ -346,25 +372,23 @@ private static ExecutorService tableExecutor(Configuration conf, int maxThreadNu
 
   /**
    * Get the committed data files for this table and job.
+   *
+   * @param numTasks Number of writer tasks that produced a forCommit file
    * @param executor The executor used for reading the forCommit files parallel
    * @param location The location of the table
    * @param jobContext The job context
    * @param io The FileIO used for reading a files generated for commit
    * @param throwOnFailure If <code>true</code> then it throws an exception on failure
    * @return The list of the committed data files
    */
-  private static Collection<DataFile> dataFiles(ExecutorService executor, String location, JobContext jobContext,
-      FileIO io, boolean throwOnFailure) {
+  private static Collection<DataFile> dataFiles(int numTasks, ExecutorService executor, String location,
+                                                JobContext jobContext, FileIO io, boolean throwOnFailure) {
```

Comment:
nit: I have seen Iceberg reviewers asking for 4 space padding.

Reply:
They are a bit confusing about this. The 4 spaces padding is indeed the general rule for line continuation, but I've been asked earlier by Anton on other PRs not to do that for method parameter indentations and do it this way.

Reply:
Fun stuff 😄
Whatever!